vividus-plugin-web-app/src/main/java/org/vividus/ui/web/action/MouseActions.java

```diff
@@ -183,8 +183,9 @@ public void moveToElement(WebElement element)
     {
         if (element != null)
         {
-            //Safari Driver doesn't scroll to element before moveTo action
-            if (webDriverManager.isMobile() || webDriverManager.isTypeAnyOf(WebDriverType.SAFARI))
+            //Safari, Firefox and Edge Drivers don't scroll to element before moveTo action
```

Comment:
// Safar

Reply:
https://github.com/checkstyle/checkstyle/issues/6593

Reply:
Still not clear for me, it just about setting some formal formatting, isn't it? :)